runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.4.2
```

Comment:
Tells Heroku and Cloud Foundry to use Python 3.4.2 as the language runtime. This is separate from specifying that it's a "Python app", which it infers from the presence of `requirements.txt`.